fabric/src/main/java/me/shedaniel/architectury/networking/fabric/SpawnEntityPacket.java

```diff
@@ -0,0 +1,67 @@
+package me.shedaniel.architectury.networking.fabric;
+
+import net.fabricmc.api.EnvType;
+import net.fabricmc.api.Environment;
+import net.fabricmc.fabric.api.client.networking.v1.ClientPlayNetworking;
+import net.fabricmc.fabric.api.networking.v1.PacketByteBufs;
+import net.fabricmc.fabric.api.networking.v1.PacketSender;
+import net.fabricmc.fabric.impl.networking.ServerSidePacketRegistryImpl;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.multiplayer.ClientPacketListener;
+import net.minecraft.core.Registry;
+import net.minecraft.network.FriendlyByteBuf;
+import net.minecraft.network.protocol.Packet;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.entity.Entity;
+import net.minecraft.world.entity.EntityType;
+
+import java.util.UUID;
+
+public class SpawnEntityPacket {
```

Comment:
This class could use the architectury NetworkManager instead for easier maintainability.

Reply:
I'm not sure how to do this and I can't find any exaple